### PR TITLE
Fix all quotes

### PR DESCRIPTION
**Pull request by szepeviktor**
**Description**
Fix all quotes

Done by a machine! (phpcbf)

**Fixed issue**
#281

@acampos1916

### CODE REVIEW

**Comment by szepeviktor** [comment]
💡 Only SonarCloud gave me an :x:

File: phpcs.xml [hunk 1]
```diff
@@ -32,4 +32,5 @@
     <arg value="p"/>
 
     <rule ref="PSR2"/>
+    <rule ref="Squiz.Strings.DoubleQuoteUsage.NotRequired"/>
```
Comment (szepeviktor) [inline phpcs.xml:35]: Could it be that it is called just `Squiz.Strings.DoubleQuoteUsage` ??

File: src/Adyen/HttpClient/CurlClient.php [hunk 2]
```diff
@@ -273,22 +273,22 @@ protected function handleCurlError($url, $errno, $message, $logger)
...
-                    . "internet connection and try again.";
+                    . 'internet connection and try again.';
                 break;
             case CURLE_SSL_CACERT:
             case CURLE_SSL_PEER_CERTIFICATE:
                 $msg = "Could not verify Adyen's SSL certificate.  Please make sure "
-                    . "that your network is not intercepting certificates.  "
+                    . 'that your network is not intercepting certificates.  '
                     . "(Try going to $url in your browser.)  "
-                    . "If this problem persists,";
```
Comment (peterojo) [inline src/Adyen/HttpClient/CurlClient.php:288]: Seems a bit strange to have different types of quotes for the same concatenated string

Comment (szepeviktor) [reply src/Adyen/HttpClient/CurlClient.php:288]: Variable do not extend in single quotes.
The default is the single quote. Double quotes are for

1. If the string contains a single quote
2. If there is a variable in it

I leave it up to you!

Quote from the top comment:

> Done by a machine! (phpcbf)

Comment (szepeviktor) [reply src/Adyen/HttpClient/CurlClient.php:288]: To be clear.
I'm not willing to contribute to quote thingies.

**Comment by szepeviktor** [comment]
💡 This may make  #289 impossible. Please do not merge it before that.

**Review by cyattilakiss — APPROVED** [review]

**Comment by AlexandrosMor** [comment]
Hello @szepeviktor,

We would like to thank you taking the time to open this pull request. This issue was solved in the later pull requests so for that reason we are going to close this pr.

kind regards,
Alexandros
Adyen

**Pull request closed without merging**